app/src/main/java/org/mozilla/fenix/share/SaveToPDFItem.kt

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.share
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * A save to PDF item.
+ * @param onClick event handler when the save to PDF item is clicked.
+ */
+@Composable
+fun SaveToPDFItem(
+    onClick: () -> Unit,
+) {
+    Row(
+        modifier = Modifier
+            .height(48.dp)
+            .fillMaxWidth()
+            .clickable(onClick = onClick),
+    ) {
+        Column {
+            Divider(
+                color = FirefoxTheme.colors.borderPrimary,
+                thickness = 1.dp,
+            )
+
+            Row(modifier = Modifier.padding(top = 16.dp)) {
+                Icon(
+                    modifier = Modifier.padding(start = 20.dp, end = 36.dp),
+                    painter = painterResource(R.drawable.ic_download),
+                    contentDescription = stringResource(
+                        R.string.content_description_close_button,
+                    ),
+                    tint = FirefoxTheme.colors.iconPrimary,
+                )
+
+                Text(
+                    stringResource(R.string.share_save_to_pdf),
+                )
+            }
+        }
+    }
```

Comment:
I noticed a few inaccuracies with the designs, so I ended updating them and adding them to this one block below:

```kotlin
    Row(
        modifier = Modifier
            .height(56.dp)
            .fillMaxWidth()
            .clickable(onClick = onClick),
    ) {
        Column {
            Divider(
                color = FirefoxTheme.colors.borderPrimary,
                thickness = 1.dp,
            )

            Row(
                modifier = Modifier.padding(top = 16.dp),
                verticalAlignment = Alignment.CenterVertically,
            ) {
                Icon(
                    modifier = Modifier.padding(start = 16.dp, end = 32.dp),
                    painter = painterResource(R.drawable.ic_download),
                    contentDescription = stringResource(
                        R.string.content_description_close_button,
                    ),
                    tint = FirefoxTheme.colors.iconPrimary,
                )

                Text(
                    stringResource(R.string.share_save_to_pdf),
                    color = FirefoxTheme.colors.textPrimary,
                )
            }
        }
    }
```

This is what it ends up looking like on my device:
<img width="409" alt="Screen Shot 2022-10-03 at 11 08 41 PM" src="https://user-images.githubusercontent.com/1370580/193726119-e717c288-4677-49f5-a9f1-991276bdf3a0.png">

Reply:
Thanks ❤️ !!